packages/frontend/editor-ui/src/components/MainHeader/WorkflowHistoryButton.vue

```diff
@@ -30,7 +30,8 @@ const workflowHistoryRoute = computed<{ name: string; params: { workflowId: stri
 			<N8nIconButton
 				:disabled="isNewWorkflow || !isFeatureEnabled"
 				data-test-id="workflow-history-button"
```

Comment:
<img width="382" height="56" alt="Image" src="https://github.com/user-attachments/assets/b7165c19-93e7-42a3-a74c-b3f764cdba60" />
The old version used to look like this. Can we have the grey background to the button.
This will also be consistent with the "ellipsis" menu on the right

Reply:
@nikhilkuria on what version are you seeing this?

Reply:
I've fixed this here: https://github.com/n8n-io/n8n/pull/18957/files

The issue was that the highlight button tokens were refrencing some outdated primitives, should be working now with these changes.

Reply:
No - but that is how they've always looked. We are reworking the buttons in the design system at the moment generally